css/dashboards/gui.css

```diff
@@ -816,3 +816,63 @@
 .highcharts-dashboards-row .highcharts-dashboards-cell:focus {
     outline: 5px auto -webkit-focus-ring-color;
 }
+
+/* Dropdown */
+ul.highcharts-dashboards-droptown-content {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+    background: #ffffff;
+    box-sizing: border-box;
+    padding: 16px 8px;
+    margin-top: 4px;
+    border: 1px solid #838394;
+    box-shadow: 0 10px 24px rgba(34, 34, 34, 0.1);
+    list-style-type: none;
+}
+
+ul > .highcharts-dashboards-dropdown {
+    position: relative;
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
Do we need to declare that font specifically to the dropdown elements?

Reply:
This is what the design says 🤷 @nasin ?

Reply:
I don't think so, let's keep the font from the main dashboard layout.